src/CommunityToolkit.Maui.MediaElement/Extensions/SubtitleExtensions.android.cs

```diff
@@ -0,0 +1,242 @@
+﻿using Android.Content.Res;
+using Android.Views;
+using Android.Widget;
+using Com.Google.Android.Exoplayer2.UI;
+using CommunityToolkit.Maui.Core;
+using CommunityToolkit.Maui.Core.Views;
+using CommunityToolkit.Maui.Primitives;
+using static Android.Views.ViewGroup;
+using Activity = Android.App.Activity;
+
+namespace CommunityToolkit.Maui.Extensions;
+
+partial class SubtitleExtensions : IDisposable
+{
+	bool disposedValue;
+
+	static readonly HttpClient httpClient = new();
+	readonly IDispatcher dispatcher;
+	readonly RelativeLayout.LayoutParams? subtitleLayout;
+	readonly StyledPlayerView styledPlayerView;
```

Comment:
This is marked as obsolete, would be easy to change it?

Reply:
If you mean `StyledPlayerView` we are waiting on `Media3` binding to update.